modules/helm-disabling-helm-chart-repository-in-the-cluster.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="disabling-helm-chart-repository-in-the-cluster_{context}"]
```

Comment:
You need to make the ID match the module name: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata